src/mesh/SX126xInterface.cpp

```diff
@@ -25,13 +25,13 @@ template <typename T> bool SX126xInterface<T>::init()
     digitalWrite(SX126X_POWER_EN, HIGH);
     pinMode(SX126X_POWER_EN, OUTPUT);
 #endif
+    // Always power on the TCXO on SX126X devices, since all of them have a TCXO onboard
+    // which needs to be activated by setting DIO3 to high. Reference:
+    // https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/blob/master/schematic/T-SX1262.pdf
 
-#ifndef SX126X_E22
-    float tcxoVoltage = 0; // None - we use an XTAL
-#else
     // Use DIO3 to power tcxo per https://github.com/jgromes/RadioLib/issues/12#issuecomment-520695575
     float tcxoVoltage = 1.8;
```

Comment:
If this place in the code is touched I suggest to make the tcxoVoltage configurable via variant files, as some boards may benefit from a different value than this default. E.g.

#ifndef TCXO_VOLTAGE
    float tcxoVoltage = 1.8;
#else
    float tcxoVoltage = TCXO_VOLTAGE;
#endif